### ISSUE TITLE
Create a calendar of events R-Ladies Global should communicate about

### ISSUE THREAD

**Issue by maelle** [rladies/rladiesguide#76]
Idea of @shannonpileggi, cc @yabellini 

- IWD
- TDOV
- The Day of Women and Girls in Science

Maybe as JSON (event name, URL, date/how to compute the date) and then input it somewhere else e.g. an online calendar + present it in the guide somewhere (as a suggestion for chapter social media too).

**Comment by maelle** [comment]
Maybe also useful for the website calendar @Athanasiamo

**Comment by shannonpileggi** [comment]
And add this to the global twitter managment guide when ready
https://github.com/rladies/rladiesguide/edit/main/content/comm/twitter/index.en.md

**Comment by shannonpileggi** [comment]
June: LGBTQIA+ Pride Month

**Comment by shannonpileggi** [comment]
adding @d4tagirl to this - the idea of this was to assist with social media content

**Comment by drmowinckels** [comment]
right, I forgot about this! Yes this is something we absolutely want to add!

If we make a json file with it, I can likely find a decent way to get it into our Events calendar on the website in another colour (so itr distinctly different from meetup events).

**Comment by nrennie** [comment]
I've started drafting a list of days/events/months that we *might* be interested in - where is the best place to add a PR for others to add their suggestions/thoughts?

**Comment by shannonpileggi** [comment]
maybe put it in the global twitter section for now, and then we create a separate issue to make that a more general social media section?? https://guide.rladies.org/comm/twitter/

would love to hear what other think as well.

**Comment by maelle** [comment]
oh right, renaming and updating the Twitter chapter would make plenty of sense :sweat_smile:

**Comment by d4tagirl** [comment]
It does! I actually read it often and I adapt it to Mastodon in my mind 🤣. I could help with this, but I will have available time in july so if anyone has the time now, I could chip in after.

**Comment by nrennie** [comment]
I've started a JSON file in #140 - please feel free to add other events that I might have missed 😄
